src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNetIntegration.cs

```diff
@@ -4,66 +4,117 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Datadog.Trace.ClrProfiler.Emit;
-using Datadog.Trace.ClrProfiler.Helpers;
-using Datadog.Trace.ExtensionMethods;
 using Datadog.Trace.Logging;
 
 namespace Datadog.Trace.ClrProfiler.Integrations
 {
     /// <summary>
-    /// AdoNetIntegration provides methods that add tracing to ADO.NET calls.
+    /// Instrumentation wrappers for <see cref="DbCommand"/>.
     /// </summary>
     public static class AdoNetIntegration
     {
+        // TODO: rename to DbCommandIntegration
         private const string IntegrationName = "AdoNet";
         private const string Major4 = "4";
-        private const string FrameworkAssembly = "System.Data";
-        private const string CoreAssembly = "System.Data.Common";
+
         private const string DbCommandTypeName = "System.Data.Common.DbCommand";
         private const string DbDataReaderTypeName = "System.Data.Common.DbDataReader";
-        private const string CommandBehaviorTypeName = "System.Data.CommandBehavior";
 
-        private static readonly ILog Log = LogProvider.GetLogger(typeof(AdoNetIntegration));
+        private static readonly ILog Log = LogProvider.GetCurrentClassLogger();
 
         /// <summary>
-        /// Wrapper method that instruments <see cref="System.Data.Common.DbCommand.ExecuteDbDataReader"/>.
+        /// Instrumentation wrapper for <see cref="DbCommand.ExecuteDbDataReader"/>.
         /// </summary>
-        /// <param name="this">The <see cref="DbCommand"/> that is references by the "this" pointer in the instrumented method.</param>
-        /// <param name="behavior">A value from <see cref="CommandBehavior"/>.</param>
+        /// <param name="command">The object referenced "this" in the instrumented method.</param>
         /// <param name="opCode">The OpCode used in the original method call.</param>
         /// <param name="mdToken">The mdToken of the original method call.</param>
         /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
         /// <returns>The value returned by the instrumented method.</returns>
         [InterceptMethod(
-            TargetAssemblies = new[] { FrameworkAssembly, CoreAssembly },
+            TargetAssemblies = new[] { AdoNetConstants.AssemblyNames.SystemData, AdoNetConstants.AssemblyNames.SystemDataCommon },
             TargetType = DbCommandTypeName,
-            TargetSignatureTypes = new[] { DbDataReaderTypeName, CommandBehaviorTypeName },
+            TargetSignatureTypes = new[] { DbDataReaderTypeName, AdoNetConstants.TypeNames.CommandBehavior },
```

Comment:
I think the command behavior type should be removed here since you removed it from the parameter list of the actual method.

Reply:
Fixed in 1bb3df0558fad485ce5b744f7ef4cdf3ad13306c.

Reply:
Can you also re-run the `GenerateIntegrationsDefinitions` tool?

Reply:
I forgot to say I regenerated in a separate commit: 4c278b9174e4202c91736dbc0a2f26d13b41776e.